grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -433,7 +433,7 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
     Integer getPostedQuantityInvoiced() {
         return allInvoiceItems?.findAll {
             it?.invoice?.datePosted != null && !it?.invoice?.isPrepaymentInvoice && !it.inverse
-        }?.collect {  it.quantity }?.sum() ?: 0
+        }?.sum { it.quantity } ?: 0
```

Comment:
I assume the quantity cannot be null right?

Reply:
Your assumption is correct

Reply:
yep